### ISSUE TITLE
[Linux] black screen on pascal w/ binary drivers

### ISSUE THREAD

**Issue by hcorion** [RPCS3/rpcs3#2578]
**Tested non-working games**
- 1942: Joint strike Demo (NPUB90104)
- Terraria
**Tested working games**
- Sogger HD (SCOG00001)
- PS3DOOM (DOOM00666) with doom2.wad
- Hero City 2 (HCTW00000)

**version**
RPCS3 v0.0.2-5035-458e5fa4 Alpha

**System specs**
Arch Linux 64-bit
NVIDIA GTX 1070
Intel i7 4770
Nvidia Driver Version: 378.13
X11 version 1.19.3
KDE Window manager
GCC version 6.3.1 20170306

**RPCS3 Settings**
PPU Decoder = Interpreter (fast)
SPU Decoder = Recompiler (ASMJIT)
Load required libraries is checked
No Load libraries are pre-selected.

**Other information**

I tried the fix  on issue #2311 and nothing changed.
I'm motivated to fix this bug.
Cmake flags:
```shell
cmake ../../source/rpcs3 -G "Unix Makefiles" -DUSE_SYSTEM_LIBPNG=ON -DUSE_SYSTEM_FFMPEG=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_EXE_LINKER_FLAGS='-ldl -lyaml-cpp' -DCMAKE_SKIP_RPATH='ON' -DLLVM_DIR=/usr/
```

**Debug log**
found in ~/.config/rpcs3/RPCS3.log
https://paste.scratchbook.ch/view/727331af

**Screenshot**
![rpcs3-buggy-screenshot](https://cloud.githubusercontent.com/assets/9814000/24278267/f27a8a2e-0fff-11e7-9ef2-a97b787e160c.jpg)

**Comment by kd-11** [comment]
```
·W 0:04:32.121635 LDR: SELF: No NPDRM control info found!
·W 0:04:32.123877 LDR: SELF: No NPDRM control info found!
```

This has less to do with linux and more with missing RAP files IMO. That NPDRM spam is not normal. Last I checked 1942 crashes the emu with linux though I haven't checked lately.

**Comment by kd-11** [comment]
Not sure if it cased by RAP. If its a graphical bug, the game should have continued working. You would hear menu music, etc and you would go ingame, just no visual output which I'm guessing is not what is happening here.

**Comment by hcorion** [comment]
Yeah, I'm not getting any music output.

 Perhaps there is a simpler Homebrew ROM you could recommend for me to try, to see if it is a RAP problem.

**Comment by kd-11** [comment]
Try scogger HD. Should work fine on linux.

**Comment by hcorion** [comment]
Ok, I'll try that tomorrow.

**Comment by al0xf** [comment]
FYI kd-11 that "No NPDRM control info found!" spam does happen on windows as well, just open settings and it gets printed to the log. 

No the issue here are not your drivers or anything but that RPCS3 on Linux is buggy in general. We already know that several games like Catherine, Demon's Souls, Persona 5 and so on do more or less nothing on Linux. It will be worked on, it is on the short term roadmap.

**Comment by hcorion** [comment]
@kd-11 
I just tested ScoggerHD, it's the first RPCS3 game that works. NPDRM spam still happening.
Log: https://paste.scratchbook.ch/view/4de4b84a

@al0xf 
I realize that RPCS3 is incredibly buggy on Linux. That's why I'm filing this bug report, so I can provide the necessary information to improve RPCS3 on linux.

**Comment by hcorion** [comment]
All my issues fixed in PR https://github.com/RPCS3/rpcs3/pull/2632

**Comment by hcorion** [comment]
Fixed by 1ae334e5006b112c20e02829af32951c613787ad
